locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nObservaci\u00f3: totes les vostres notes seran enviades a la paperera.
```

Comment:
@dylano ... i really pasted EXACTLY the characters you pasted (you know: copy&paste). In fact, you can see also in this screenshot of my editor, that it's seing fine in it. 

I must remember to you that as i explained a month ago in my first pull request: I AM NOT ABLE to get that characters well pulled/pasted in the code of this app... I can assure you that i've pushed translations to other apps, and this time is the first i cannot and i don't know how to get it done! :(

I accept suggestions. Cannot to paste yourself ?
![2016-01-15-17-55-21](https://cloud.githubusercontent.com/assets/1767547/12369275/ca697a7a-bbb1-11e5-93b9-259dc8528b5e.png)